weaviate/batch/requests.py

```diff
@@ -214,7 +214,7 @@ def add(self,
         if uuid is not None:
             batch_item["id"] = get_valid_uuid(uuid)
         else:
-            batch_item["id"] = uuid4().hex
+            batch_item["id"] = get_valid_uuid(uuid4().hex)
```

Comment:
Since the `get_valid_uuid` function already does the validation, I would pass only the returned value of the `uuid4()`:
```python
batch_item["id"] = get_valid_uuid(uuid4())
```
Or you could leave it as it was before (`batch_item["id"] = uuid4().hex`), because that one is definitely a valid one.